stripe/src/main/java/com/stripe/android/StripeResponse.java

```diff
@@ -37,6 +38,10 @@ int getResponseCode() {
         return mResponseCode;
     }
 
+    boolean isSuccessful() {
+        return mResponseCode == HttpURLConnection.HTTP_OK;
```

Comment:
Should we also range check for is successful like error below?

From https://stripe.com/docs/api/errors `Codes in the 2xx range indicate success`

Reply:
I'll rename to "isOk" to be more explicit